src/containers/AdminQuestionsContainer.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { IQuestion } from '../interfaces';
+import {
+  getQuestionList,
+  dateStringFormat,
+  studentGradeFormat,
+} from '../services';
+
+const AdminQuestionsContainer = () => {
+  const [publicQuestionList, setPublicQuestionList] = useState(
+    [] as IQuestion[],
+  );
+
+  useEffect(() => {
+    getQuestionList<IQuestion[]>('public').then(setPublicQuestionList);
+  }, []);
+
+  return (
+    <div>
+      <div className="container--header">
+        <h3>Alle godkjente spørsmål</h3>
+      </div>
+      <div className="admin-questions">
+        <div className="admin-questions--item">
+          <h5>Publisert</h5>
+          <div className="admin-questions--list">
+            {publicQuestionList.length > 0 ? (
+              publicQuestionList.map((question, index) => {
```

Comment:
Ta dette ut i en egen funksjon? Kan også bruke useMemo for å slippe å gjøre utregningene her hver render. Typ: `const questionList = useMemo(() => (*stuff to return*), [])`